filesystems/blob/src/main/scala/cromwell/filesystems/blob/BlobFileSystemManager.scala

```diff
@@ -86,8 +86,8 @@ object BlobTokenGenerator {
   def createBlobTokenGenerator(container: BlobContainerName, endpoint: EndpointURL, subscription: Option[SubscriptionId]): BlobTokenGenerator = {
```

Comment:
I think this could use some comments around why we have these two types of SAS token acquisition strategy, and what they're used for. Or maybe it's a pointer to a longer comment elsewhere. With two different types of "tokens" and multiple ways of obtaining each, this gets pretty complex.